{
  "name": "wait-for-throwable",
  "version": "1.0.0",
  "description": "Simple utility to retry an erroring function until it succeeds",
  "main": "wait-for-throwable.js",
  "dependencies": {},
  "devDependencies": {},
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/catdad/wait-for-throwable.git"
  },
  "keywords": [],
  "author": "Kiril Vatev <vatev.1@gmail.com>",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/catdad/wait-for-throwable/issues"
  },
  "homepage": "https://github.com/catdad/wait-for-throwable#readme"
}
